app/controllers/hosts_controller.rb

```diff
@@ -68,6 +68,8 @@ def show
 
         # summary report text
         @report_summary = ConfigReport.summarise(@range.days.ago, @host)
+
+        @active_tab = (params["active-tab"].empty? ? 'properties' : params["active-tab"])
```

Comment:
```suggestion
        @active_tab = params["active-tab"] || 'properties'
```
unless the value defaults to `''`